### DISCUSSION TITLE
Can Ruby 2D work with RVM instead of the recommended rbenv?

### DISCUSSION THREAD

**Discussion by OZcomingFRoo** [ruby2d/ruby2d#223] [Q&A] [answered]
Since I heard they [can't coexist together](https://stackoverflow.com/questions/35808103/rvm-and-rbenv-on-the-same-machine), I need to know if I can work with the RVM instead

**Comment by blacktm** [comment, marked as answer]
Yes, should work with all Ruby version mangers.
